### ISSUE TITLE
Sort node is extremely slow in listview with many children

### ISSUE THREAD

**Issue by bjarnef** [umbraco/Umbraco-CMS#13490]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

10.3.2

### Bug summary

We have a project on Umbraco v10.3.2 with a listview with ~30K nodes, but when opening sort dialog this has some performance issues. After 1.5 minutes it is still pending to load nodes.

![image](https://user-images.githubusercontent.com/2919859/204306123-9195e7b2-d578-4942-8ffe-db020793cb3a.png)

### Steps to reproduce

Create a listview with 10K+ nodes and open sort dialog.
This should take a while, probably +1 minutes.

**Comment by nzdev** [comment]
I wonder if it's even practical to be sorting that many nodes. 
I think once this sort of scale is reached it may be better to not have a sort at all. With the block list available, situations that require a sorted list of items may be better as a single node containing the block lists and the subset needed for sort.

**Comment by Zeegaan** [comment]
I can definitely reproduce this 😁 
With 8k nodes the call is just 2 seconds to the database, but the frontend cannot handle that many nodes, and just crashes on me 🤔

**Comment by nzdev** [comment]
Thinking about this, the better option for Umbraco CMS would be for the sortOrder property to be removed from the CMS by the time there is a new backoffice so this situation cannot occur as I don't think there is a good UX solution for sorting 30k items. 
It's also something that slows down creating / updating content as changing sort order affects nucache and examine, same as the tree structure itself.

In the meantime, a reasonable solution would be to not rely on the sortOrder and use a dedicated content node with a property that allows content editors to pick and sort content such as any of the content pickers, or block lists. This is what I do for a 300k+ node site.

**Comment by bjarnef** [comment]
@Zeegaan yes, the browser may be slow/crash when Angular tries to render more than 500-1000 nodes. I paged result would probably improve this, but not sure if sorting 8K nodes itself would be slow.

The sort action can probably not be removed as many may use it for fewer nodes, where it may work well.

**Comment by mayhammf** [comment]
(Not sure this is entirely relevant but it's the closest issue I could find)

There is a serious performance issue when it comes to rendering (even small lists) in Umbraco 10 compared with Umbraco 7. We noticed this as we are migrating custom property editors. 
One of the cases we have is a property editor with 250 items including checkboxes. When rendering the items via ng-repeat, the browser freezes for about 1 second.
We did some profiling and the main chocking seems to be caused by the disabletabindex directive which watches all DOM changes on every tabbable element to try and set the tabindex attribute to -1.

https://github.com/umbraco/Umbraco-CMS/blob/8c2329f2b20a6227f586973d32349321c4e81529/src/Umbraco.Web.UI.Client/src/common/directives/util/disabletabindex.directive.js

I will create a detailed bug with repro steps.

**Comment by mayhammf** [comment]
Issue here: #13544
